server/channels/app/channel_test.go

```diff
@@ -269,17 +294,18 @@ func TestJoinDefaultChannelsCreatesChannelMemberHistoryRecordTownSquare(t *testi
 	th := Setup(t).InitBasic()
 	defer th.TearDown()
 
-	// figure out the initial number of users in town square
+	// figure out the initial number of testUsers in town square
```

Comment:
That doesn't seem correct. `users` is a fine term.

Reply:
My IDE was complaining this is clashing with the imported users package. Though both `make golangci-lint` and `make vet` are happy with, so I'll revert this.